session_exist_test.go

```diff
@@ -73,4 +73,8 @@ func TestExistStruct(t *testing.T) {
 	has, err = testEngine.Table("record_exist").Where("name = ?", "test2").Exist()
 	assert.NoError(t, err)
 	assert.False(t, has)
+
+	has, err = testEngine.Table("record_exist").ID(100).Where("name = ?", "test1").Exist()
+	assert.NoError(t, err)
```

Comment:
```Go
assert.Error(t, err)
```

Reply:
`Table("record_exist")` with `ID` won't be supported?

Reply:
Yes, xorm don't know which column is primary key column since no struct was indicated.